public/ckeditor/samples/old/assets/posteddata.php

```diff
@@ -26,24 +26,24 @@
 		</thead>
```

Comment:
This file was vendor-provided. Do you think we should re-format this code?

Reply:
Yeah this should be ignored. I'll modify the config.

Reply:
Why is ckeditor not ignored by version control? Is there a specific reason or was it just not added? Ideally, that entire directory should be ignored from version control if it is truly vendor provided (then we wouldn't have to worry about the formatting).

Reply:
Nevermind ignore that last one. I looked at it and I remember doing this. We probably should write a story to install this with npm (and upgrade to v5 while we're at it) instead of the manual install like what I did. There's no way to update this and there's no clue how many vulnerabilities exist with it at this point in time. It's only used in the admin panel so it's probably not a high priority, but we definitely should do it.

I'll add this directory to be ignored from the formatting for now, and then write another story to make that change.

Reply:
Resolved in [b1e736d](https://github.com/ZTL-ARTCC/Current_Website/pull/187/commits/b1e736d41d7f295dab7e57fac3a623ecb9e066db) and [c3ed659](https://github.com/ZTL-ARTCC/Current_Website/pull/187/commits/c3ed659af3cd5c5e26ed4a128eca208a38a83f84). Will link the story to include this with npm momentarily.